src/helper_classes/HttpArguments.java

```diff
@@ -0,0 +1,74 @@
+package helper_classes;
```

Comment:
Please follow Jave style guide for naming convention. Refer https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Reply:
done